### DISCUSSION TITLE
Deleting an event on google calendar. Is there a way to stop resync/repopulation of event?

### DISCUSSION THREAD

**Discussion by peliwagner** [derekantrican/GAS-ICS-Sync#431] [Q&A]
I am trying to figure out if its possible to delete one of the ICS events from my google calendar and prevent the repopulation everytime the script runs. Is this even possible?

**Comment by jonas0b1011001** [comment]
You would have to filter the event from within the script. see #376

**Comment by jonas0b1011001** [comment]
I assume this is resolved/not longer relevant. Closing.
